pkg/cargo/files.go

```diff
@@ -20,6 +20,8 @@ func InterpolateAndParseKilnfile(in io.Reader, templateVariables map[string]any)
 		return Kilnfile{}, fmt.Errorf("unable to read Kilnfile: %w", err)
 	}
 
+	fmt.Println(string(kilnfileYAML))
```

Comment:
Did you mean to remove this print statement?

Reply:
Yes. My bad. Thanks for catching this.